Scripts/script-Sleep.yml

```diff
@@ -16,5 +14,4 @@ args:
   default: true
   description: Amount of seconds to sleep
 scripttarget: 0
-dependson: {}
-timeout: 6000s
+timeout: 1h40m0s
```

Comment:
It was 6000s in the Python version (I don't know why - I think it's arbitrary), which is 1 hour and 40 minutes.